### DISCUSSION TITLE
Welcome to ABAQUS_subroutines Discussions!

### DISCUSSION THREAD

**Discussion by davidmorinNTNU** [davidmorinNTNU/ABAQUS_subroutines#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ariellusty** [comment]
Greetings, thank you for sharing a umat example. When I try to run it, though, there's an error: "4 nodes have dof on which incorrect boundary conditions may have been specified. The nodes have been identified in node set WarnNodeBCIncorrectDof." There are 13 boundary conditions, so it's tough to figure out which one's off. Any insight as to what might be happening?

**Comment by qll007** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    不好意思，我也不懂
    
        
            
        
    
    
     
    
        
                    ***@***.***
        
    
 

    On 6/16/2022 ***@***.***> wrote： 



Greetings, thank you for sharing a umat example. When I try to run it, though, there's an error: "4 nodes have dof on which incorrect boundary conditions may have been specified. The nodes have been identified in node set WarnNodeBCIncorrectDof." There are 13 boundary conditions, so it's tough to figure out which one's off. Any insight as to what might be happening?

**Comment by ch9411** [comment]
Dear Davidmorin，
        Thank you for sharing ABAQUS_subroutines. I would like to ask for your help here. I would like to use abaqus for THMC multi-field coupling is this possible. Can you give me some advice?
